app/es_indices/observation_index.rb

```diff
@@ -240,12 +250,26 @@ def self.prepare_batch_for_index(observations, options = {})
     # fetch all project_ids store them in `indexed_project_ids`
     if options.blank? || options[:projects]
       connection.execute("
-        SELECT observation_id, project_id, curator_identification_id, uuid, user_id
```

Comment:
The changes here can be removed too. You won't need to change anything in `observation_index`